emcc.py

```diff
@@ -2002,6 +2002,11 @@ def check_memory_setting(setting):
     settings.MINIFY_WASM_IMPORTS_AND_EXPORTS = 1
     settings.MINIFY_WASM_IMPORTED_MODULES = 1
 
+  # Cache the wasm table for speed in builds not optimizing for size. We also
+  # cannot cache when using dynamic linking.
+  settings.CACHE_WASM_TABLE = not settings.RELOCATABLE and \
```

Comment:
I think we can probably make it work for RELOCATABLE too because the table mods are all centrally performed.   Happy to see this land as is though.